litecommands-sponge/src/dev/rollczi/litecommands/sponge/LiteSpongeFactory.java

```diff
@@ -27,6 +27,12 @@ public static <B extends LiteCommandsBuilder<CommandCause, LiteSpongeSettings, B
         return (B) LiteCommandsFactory.builder(CommandCause.class, new SpongePlatform(plugin, settings)).selfProcessor((builder, internal) -> {
             MessageRegistry<CommandCause> messageRegistry = internal.getMessageRegistry();
 
+            if (game.isServerAvailable()) {
```

Comment:
Please, move this to constructor, (and extract to private method and throw when server and client is not available, ISE)